components/event-service/cmd/event-service/commands/serve.go

```
@@ -36,7 +37,7 @@ func newServeCmd() *cobra.Command {
 
 			closer, err := tracing.NewGlobalTracer("event-service")
 			if err != nil {
-				errors.Wrap(err, "starting tracer for event-service")
+				return errors.Wrap(err, "starting tracer for event-service")
```

This is the only meaningful change.  Previous we were ignoring this error.